submissions/AriadnaKravchuk/frogger-game/app.js

```diff
@@ -0,0 +1,209 @@
+const SKIN = {
+    GIRL: "images/char-cat-girl.png",
+    BOY: "images/char-boy.png"
+};
+
+const SPEED = {
+    MIN: 400,
+    MAX: 700
+};
+
+const STEP = {
+    X: 100,
+    Y: 80,
+    HALF_X: 50,
+    HALF_Y: 40,
+};
+
+const BORDER = {
+    MIN_X: 0,
+    MAX_X: 400,
+    MIN_Y: 375,
+    MAX_Y: -25
+};
+
+const PLAYER_POSITION = {
+    START_X: 200,
+    START_Y: 375
+};
+
+const TIME_TIMEOUT = 200,
+    ALL_X = [
+        BORDER.MIN_X,
+        BORDER.MIN_X + STEP.X,
+        BORDER.MIN_X + STEP.X * 2,
+        BORDER.MIN_X + STEP.X * 3,
+        BORDER.MAX_X,
+    ],
+    ENEMIES_Y = [
+        BORDER.MAX_Y + STEP.Y * 3, 
+        BORDER.MAX_Y + STEP.Y * 2, 
+        BORDER.MAX_Y + STEP.Y
+    ],
+    ROCKS_Y = [
+        BORDER.MAX_Y,
+        BORDER.MAX_Y,
+        BORDER.MAX_Y,
+        BORDER.MAX_Y
+    ]
+
+let winScore = 0,
+    loseScore = 0,
+    allEnemies = [],
+    allRocks = [];
+
+function getNumFromArea(min, max) {
+    return Math.random() * (max - min) + min;
+}
+
+Array.prototype.getNumFromArray = function () {
+    return this[Math.floor(Math.random() * this.length)];
+};
+
+function changeChar(player) {
```

Comment:
Avoid abbreviations. We write variable/function names once, then IDE helps to autcomplete.
Does `Char` mean `character`, `charcoal`, `Charizard`? (a rhetoric question)